scripts/github_actions/generate_build_matrix.py

```diff
@@ -284,12 +284,12 @@ def generate_build_matrix(
         # https://github.com/Jimver/cuda-toolkit/blob/master/src/links/windows-links.ts
     }
     if test_only_latest_torch:
-        latest = "2.5.0"
+        latest = "2.5.1"
         matrix = {latest: matrix[latest]}
 
     if for_windows or for_macos:
-        if "2.5.0" in matrix:
-            matrix["2.5.0"]["python-version"].remove("3.13")
+        if "2.5.1" in matrix:
```

Comment:
Please keep the original `2.5.0` and you can add a new test for `2.5.1`

Reply:
done